### PR TITLE
Support multiple search (string / text / belongs_to_association)

### PR DESCRIPTION
**Pull request by sltc-li**

### CODE REVIEW

**Comment by denvazh** [comment]
LGTM!
[LGTM](http://lgtm.in/i/eFVtPkR8q)

[:+1:](http://lgtm.in/u/eFVtPkR8q) [:-1:](http://lgtm.in/r/eFVtPkR8q)

File: app/assets/javascripts/rails_admin/ra.filter-box.js [hunk 1]
```diff
@@ -69,15 +69,18 @@
...
-            '<option data-additional-fieldset="additional-fieldset"'  + (field_operator == "ends_with"   ? 'selected="selected"' : '') + ' value="ends_with">' + RailsAdmin.I18n.t("ends_with") + '</option>' +
+            '<option data-additional-fieldset="in"'           + (field_operator == "in"          ? 'selected="selected"' : '') + ' value="in">' + RailsAdmin.I18n.t("in") + '</option>' +
+            '<option data-additional-fieldset="like"'         + (field_operator == "like"        ? 'selected="selected"' : '') + ' value="like">' + RailsAdmin.I18n.t("contains") + '</option>' +
+            '<option data-additional-fieldset="is"'           + (field_operator == "is"          ? 'selected="selected"' : '') + ' value="is">' + RailsAdmin.I18n.t("is_exactly") + '</option>' +
+            '<option data-additional-fieldset="starts_with"'  + (field_operator == "starts_with" ? 'selected="selected"' : '') + ' value="starts_with">' + RailsAdmin.I18n.t("starts_with") + '</option>' +
+            '<option data-additional-fieldset="ends_with"'    + (field_operator == "ends_with"   ? 'selected="selected"' : '') + ' value="ends_with">' + RailsAdmin.I18n.t("ends_with") + '</option>' +
```
Comment (Jwata) [inline app/assets/javascripts/rails_admin/ra.filter-box.js:76]: btw why were the other lines changed ?

Comment (liyy2088) [inline app/assets/javascripts/rails_admin/ra.filter-box.js:76]: @Jwata 
It doesn't have show/hide for this section as it had only one `input` field, now it has `textarea` and `input`, show/hide becomes required.

**Comment by Jwata** [comment]
LGTM!
[LGTM](http://lgtm.in/i/OrzaTOtPn)

[:+1:](http://lgtm.in/u/OrzaTOtPn) [:-1:](http://lgtm.in/r/OrzaTOtPn)

**Pull request merged**